backend/src/routes/group.ts

```diff
@@ -23,30 +22,26 @@ router.get('/:grouphandle', async (ctx, next) => {
 	ctx.body = group;
 });
 
-//Create a new group with required group handle and display name.
-router.put('/:grouphandle/:displayname/:creatorhandle', async (ctx, next) => {
-	// once auth is set up, fix this to use object id provided by auth info.
+router.put('/:grouphandle', async (ctx, next) => {
 	const user = await User.findOneBy({
-		handle: ctx.params.creatorhandle,
+		username: ctx.params.creatorhandle,
 	});
 	if (user === null) {
 		console.log('This user does not exist.');
 		ctx.status = 404;
 		return;
 	}
 	const group = new Group();
-	group.handle = ctx.params.grouphandle;
-	group.displayname = ctx.params.displayname;
-	//Create new user array for followers and members.
+	group.handle = ctx.state.user.handle;
```

Comment:
only have access to user obj id or netid through ctx.state.user - can use the obj id to find the user model and then get the other fields

Reply:
also, handle is now username!